clang/include/clang/Basic/AttrDocs.td

```diff
@@ -5557,6 +5557,21 @@ See :doc:`LTOVisibility`.
   }];
 }
 
+def TypeVisibilityDocs : Documentation {
+  let Category = DocCatType;
+  let Content = [{
+The ``type_visibility`` attribute allows the ELF visibility of a type and its vague
```

Comment:
Does this only impact ELF or does this also impact say PE32?

Reply:
I don't know the answer to that question since I have no familiarity with PE32. However, looking at the original patch it looks to me like this impacts all places where a normal visibility attribute is relevant. I think I should just remove `ELF` here from the docs, since this makes it look unnecessarily specific to ELF.